tabCount Extension/SafariExtensionViewController.swift

```diff
@@ -186,9 +186,9 @@ class SafariExtensionViewController: SFSafariExtensionViewController, NSTextFiel
             switch res {
             case .success(let counts):
                 
-                let attrs1 = [NSAttributedString.Key.foregroundColor : NSColor.black]
-                let attrs2 = [NSAttributedString.Key.foregroundColor : NSColor.darkGray]
-                let attrs3 = [NSAttributedString.Key.font : NSFont.systemFont(ofSize: 16), NSAttributedString.Key.foregroundColor : NSColor.darkGray]
+                let attrs1 = [NSAttributedString.Key.foregroundColor : NSColor.labelColor]
```

Comment:
Those changes make sense. Please be aware that this extension is no longer in the app store, so you will need to compile yourself in XCode

Reply:
I have done so, cheers! Hope you are well.